src/actions/guides/testing/introduction.cr

```diff
@@ -0,0 +1,75 @@
+class Guides::Testing::Introduction < GuideAction
+  guide_route "/testing/introduction"
+
+  def self.title
+    "Intro to Testing"
+  end
+
+  def markdown : String
+    <<-MD
+    ## Getting Started
+
+    All of your tests will go in the `spec/` directory located at the top level of your application.
+    When you generate your Lucky project, there are a few files generated for you to help make testing
+    easier.
+
+    > The files generated will be different based on Full/Api and/or Authentication generated.
+
+    ### spec_helper.cr
+
+    The `spec/spec_helper.cr` file will require all of the necessary files and configuration your testing
```

Comment:
Maybe this should say `spec suite`? I have trouble deciding when to call them "tests" vs when to call them "specs". I always try to keep my students in mind that have 0 background in any of this, and things I don't want to confuse new comers.

Reply:
This is tricky. In reality I would love to change any instance of spec to test. Including the folder and helpers, but we of course can't do that :P

I think we should be consistent and use `spec` in the guides. It is hard since I refer to them as tests, but consistency is good. Maybe a small section in the intro like "Specs and tests, what's the difference?" And explain some people have specific ways of using them but in general they are interchangeable. Crystal uses `spec` and that's what we use in the guide...or something like that